apps/backoffice/content/static/skills/workflows/SKILL.md

```
@@ -0,0 +1,75 @@
+  });
+
+  const approval = await step.waitForEvent("approval", {
+    type: "approval",
+    timeout: "15 minutes",
+  });
+
+  return { request, approval: approval.payload };
+});
+```
+
+    Put side effects, provider calls, and expensive work inside `step.do`. Keep pure deterministic
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)